lib/slackify/router.rb

```diff
@@ -23,10 +26,74 @@ def call_command(message, params)
 
           Slackify.configuration.unhandled_handler.unhandled(params)
         else
-          new_params = params.merge(command_arguments: command.regex.match(message).named_captures)
+          new_params = params.merge({ command_arguments: extract_arguments(message, command) })
```

Comment:
Could we drop the `{}` or this would fail lint?

Reply:
Yup, easy to drop, I personally prefer the explicitness to people that are new to ruby, but I switched it over